src/pages/guides/endpoints/classifications/import-file.md

```diff
@@ -104,17 +104,17 @@ curl -X POST "https://analytics.adobe.io/api/{GLOBALCOMPANYID}/classification/jo
 
 The example request above creates an import job with the following specifications:
 
-* the `dataFormat` for the classification as `tsv`.
-* the `jobName` to be `testsuite evar1 classifications`.
-* the notification will be delivered by `email` to `john@example.com` when the state is `completed`.
-* the data source is `Direct API Upload`.
```

Comment:
These are independent clauses, each completing the sentences set up by the independent introductory clause.